src/Command/PullRequest/PullRequestCloseCommand.php

```diff
@@ -47,11 +48,16 @@ protected function configure()
      */
     protected function execute(InputInterface $input, OutputInterface $output)
     {
-        $prNumber = $input->getArgument('pr_number');
-        $closingComment = $input->getOption('message');
-
         $adapter = $this->getAdapter();
+        $prNumber = $input->getArgument('pr_number');
+        $pr = $adapter->getPullRequest($prNumber);
+        $authenticatedUser = $this->getParameter($input, 'authentication')['username'];
+        $removeSourceBranch = $input->getOption('remove-source-branch');
```

Comment:
Maybe do a `strtolower` here, just for safety sake for the checks below. Or maybe the option should be changed to not take a value, so it is set to `true` when passed through